src/components/ec-alert/ec-alert.spec.js

```diff
@@ -0,0 +1,114 @@
+import Vue from 'vue';
+import { mount } from '@vue/test-utils';
+import EcAlert from './ec-alert.vue';
+import { withMockedConsole } from '../../../tests/utils/console';
+
+describe('EcAlert', () => {
+  function mountAlert(props, mountOpts) {
+    return mount(EcAlert, {
+      propsData: {
+        title: 'Title example',
+        type: 'info',
+        ...props,
+      },
+      ...mountOpts,
+    });
+  }
+
+  it('should throw if no props were given', () => {
+    withMockedConsole((errorSpy) => {
+      mount(EcAlert);
+      expect(errorSpy).toHaveBeenCalledTimes(2);
+      expect(errorSpy.mock.calls[0][0]).toContain('Missing required prop: "type"');
+      expect(errorSpy.mock.calls[1][0]).toContain('Missing required prop: "title"');
+    });
+  });
+
+  it('should display only with a title given and the type', () => {
+    const wrapper = mountAlert({ title: 'Random Title', type: 'info' });
```

Comment:
can we keep consistency with line breaks in specs? sometimes we use blank lines, sometimes we don't

Reply:
My bad I just start to use the lint extension for VC and it confused me :S

Reply:
Done!